src/sentry/integrations/vsts/integration.py

```diff
@@ -292,15 +291,15 @@ def build_integration(self, state):
         data = state['identity']['data']
         oauth_data = self.get_oauth_data(data)
         account = state['account']
-        instance = state['instance']
         user = get_user_info(data['access_token'])
         scopes = sorted(VSTSIdentityProvider.oauth_scopes)
+        base_url = self.get_base_url(data['access_token'], account['accountId'])['locationUrl']
```

Comment:
Can `get_base_url` just return `...['locationUrl']` if that's where the Base URL lives?